Dockerfile

```diff
@@ -0,0 +1,42 @@
+# created by Naga Ravi Chaitanya Elluri
+# Edited by Naga Ravi Chaitanya Elluri
+
+FROM ubuntu:latest
+MAINTAINER nelluri@deis.com
+RUN apt-get update
+RUN apt-get -y install make python lsb-release unzip wget vim git curl
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.0.1/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+WORKDIR /home
+RUN wget --no-check-certificate https://dl.bintray.com/deis/helm/helm-v0.1.0+c600ce8-linux-amd64.zip
+RUN unzip helm-v0.1.0+c600ce8-linux-amd64.zip
+RUN mv helm /usr/local/bin/
+RUN apt-get install jq
+ENV CLOUD_SDK_REPO cloud-sdk-precise
+RUN echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee /etc/apt/sources.list.d/google-cloud-sdk.list
+RUN curl -Lk https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
```

Comment:
Use curl without the -k (because we should be relying on certificates, and if we can't we should figure out why we can't)